.circleci/config.yml

```diff
@@ -582,6 +582,63 @@ jobs:
           working_directory: /pika/build
           when: always
 
+  arm64_build:
+    machine:
+      image: ubuntu-2004:current
+    resource_class: arm.large
+    steps:
+      - checkout
+      - run:
+          name: Building Arm64
+          command: |
+            # Check architecture = arm
+            uname -m
+            # Install CMake
+            wget -q https://cmake.org/files/v3.22/cmake-3.22.6-linux-aarch64.tar.gz \
+              -O cmake.tar.gz && echo \
+              "79be85d3e76565faacd60695cee11d030f7e7dd8691476144fa25eb93dbd0397  cmake.tar.gz" \
+              | sha256sum --check --status && \
+              sudo tar --strip-components=1 -C /usr/local -xf cmake.tar.gz && \
+              rm cmake.tar.gz
+            mkdir build
+            git clone https://github.com/pika-org/pika.git
```

Comment:
This should be removed and use the `checkout` step instead.

Reply:
What exactly do you mean by it doesn't work? The directory is not visible after cloning? If it doesn't work this will have to be changed to actually check out the branch that triggers the build.